test.js

```diff
@@ -12,14 +12,32 @@ tape('unknown', function (t) {
 })
 
 tape('txt', function (t) {
-  testEncoder(t, packet.txt, Buffer.allocUnsafe(0))
```

Comment:
Why remove this test?

Reply:
Empty buffer is not allowed.

Reply:
Length of zero has to be supported. From RFC 6763:

> The format of each constituent string within the DNS TXT record is a single length byte, followed by 0-255 bytes of text data.

Reply:
RFC 6763 says there is a length byte (which could be zero) followed by data. That means there is a minimum of 1 byte. I think a zero length buffer should still be a test case but the library should handle this error correctly.

Reply:
> there is a length byte (which could be zero) followed by data. That means there is a minimum of 1 byte

Pretty sure the length byte itself should not count as data, so a length byte of 0x00 followed by no data is still valid.

Reply:
Yes, I agree.

Reply:
@silverwind @pusateri Both of you are correct. 

 I was mistaken in thinking that the array of strings is teminated by NUL byte (length 0).  This is wrong.  Will update the code.